index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="AppBar">
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
+          </div>
+        </header>
+      </div>
+      <div class="headline">
+        <div class="content">
+          <div class="PrimaryText">
+            <h1 class="PrimaryText">Hey There. I'm faishar</h1>
+          </div>
+          <div class="SupportingText">
+            <p class="SupportingText">I'm a Software Developer</p>
```

Comment:
Kindly make sure that the subtitle uses a header tag `hx (examples: <h2>,<h3>, etc...)` 🤗

- This way will be semantically correct and the screen readers and other technologies will thank you.